client/src/components/advancedSelectWidget/AdvancedSelect.js

```diff
@@ -460,6 +486,7 @@ const AdvancedSelect = ({
     setPageNum(results && filterResults ? filterResults.pageNum : 0)
     setIsLoading(shouldFetchResults)
     setFetchType(shouldFetchResults ? FETCH_TYPE.NORMAL : FETCH_TYPE.NONE)
+    setShowCreateEntityComponent(false)
```

Comment:
This line is unnecessary; `showCreateEntityComponent` will always already be `false` here.